includes/class-sensei-grading.php

```diff
@@ -227,8 +245,27 @@ public function grading_default_view() {
 			?>
 		</form>
 		<?php
+		/**
+		 * Fires after the container with main content on the Grading page.
+		 *
+		 * @hook grading_after_container
```

Comment:
```suggestion
		 * @hook sensei_grading_extra
```

Is the description up-to-date as well?